### DISCUSSION TITLE
Pull Request Spam

### DISCUSSION THREAD

**Discussion by SethClydesdale** [community/community#22804] [New to GitHub]
<p>Is anyone else having random accounts open unnecessary or spam-ish pull requests in their repos? It started for me today around 4 hours ago and they just seem to keep coming.</p>
<p>I’ve marked the suspected PRs here:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/SethClydesdale/genki-study-resources/pulls?q=label:invalid" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/10122549/181073180-839c99d8-5c53-4853-b66a-f71bcf1cd8ce.png" class="thumbnail onebox-avatar" width="250" height="250">

<h3><a href="https://github.com/SethClydesdale/genki-study-resources/pulls?q=label:invalid" target="_blank" rel="noopener nofollow ugc">SethClydesdale/genki-study-resources</a></h3>

<p>A collection of exercises for practicing what is taught in Genki: An Integrated Course in Elementary Japanese. - SethClydesdale/genki-study-resources</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Most of these accounts seem like bots, too. What’s the deal with this?</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/SethClydesdale">@SethClydesdale</a>, thank you for being here! We are looking into ways that we can help make dealing with spam more effective for maintainers. As of now, you’d have to report specific content on GitHub, to <a href="https://github.com/contact">https://github.com/contact</a> with the info for us to investigate, and help. I’m sorry I don’t have a better answer as of now :slightly_frowning_face:</p>

**Comment by SethClydesdale** [comment]
<p>Ah, that’s good to hear you’re looking into ways to manage it. I suppose for now I’ll just handle it as I have been, if anymore comes my way. Thankfully no more at the moment.</p>
<p>Also just realized it’s October, so I’m assuming this influx is related to Hacktoberfest.</p>
<p>Thanks for the reply. :+1:</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Check this out, it might help: <a href="https://twitter.com/github/status/1311772722234560517">https://twitter.com/github/status/1311772722234560517</a></p>

**Comment by kingthorin** [comment]
<p>Likely related to Hacktoberfest. Please make sure you close them and apply the <code>invalid</code> label.</p>

**Comment by SethClydesdale** [comment]
<aside class="quote group-github-employees" data-username="AndreaGriffiths11" data-post="4" data-topic="135347" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 AndreaGriffiths11:</div>
<blockquote>
<p>Check this out, it might help: <a href="https://twitter.com/github/status/1311772722234560517" rel="noopener nofollow ugc">https://twitter.com/github/status/1311772722234560517 </a></p>
</blockquote>
</aside>
<p>Nice! Might have to try that if it gets worse.</p>
<aside class="quote no-group" data-username="kingthorin" data-post="5" data-topic="135347" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 kingthorin:</div>
<blockquote>
<p>Please make sure you close them and apply the <code>invalid</code> label.</p>
</blockquote>
</aside>
<p>Will do! :+1:</p>

**Comment by SethClydesdale** [comment]
<p>After today’s load of spam – including one user who opened 4 separate pull requests for 4 unnecessary changes – I decided to enable that new feature for a month. *sigh* It’s a shame too, because I honestly welcome any improvements that are meaningful, but this Hacktoberfest spam is nothing of the sort.</p>
<p>Just wanted to say thank you for adding this! It’s unfortunate I have to actually use it, but at least now I can get some peace and quiet for the rest of this month while I redirect real feedback to other channels.</p>

**Comment by kingthorin** [comment]
<p>That is really unfortunate. Luckily the projects I’m involved with have only seen one spam PR so far (fingers crossed).</p>
<p>Sad that people are so desperate for a T-Shirt and some stickers.</p>
<p>Really unless they’ve just bot’d the submission of useless changes it isn’t that hard or time consuming to actually do something of value for the Open Source community.</p>

**Comment by LWFlouisa** [comment]
This recently started happening and had to close my stuff to contributers only.

**Comment by dohyeon5626** [comment]
Most solutions seem to handle spam after the PR is already created, which still leaves a lot of noise.

I tried a simple approach — adding a verification step after a PR or issue is created — just to see if it could reduce some of it.

https://github.com/dohyeon5626/bot-check-action
